src/quo2/components/profile/collectible/view.cljs

```diff
@@ -0,0 +1,99 @@
+(ns quo2.components.profile.collectible.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.profile.collectible.style :as style]
+            [react-native.core :as rn]))
+
+(defn remaining-tiles
+  [amount]
+  [rn/view {:style (merge style/bottom-right (style/remaining-tiles))}
+   [text/text
+    {:style  (style/remaining-tiles-text)
+     :size   :paragraph-2
+     :weight :medium}
+    (str "+" amount)]])
+
+(defn tile
+  [{:keys [style resource size]}]
+  (let [source (cond->> resource
+                        (string? resource)
+                        (hash-map :uri))]
+    [rn/view {:style style}
+     [rn/image
+      {:style  (style/tile-style-by-size size)
+       :source source}]]))
+
+(defn two-tiles
+  [{:keys [images size]}]
+  [:<>
+   [tile
+    {:style    style/top-left
+     :size     size
+     :resource (first images)}]
+   [tile
+    {:style    style/bottom-right
+     :size     size
+     :resource (second images)}]])
+
+(defn three-tiles
+  [{:keys [tiles size]}]
+  (let [[image-1 image-2 image-3 & _] tiles]
+    [:<>
+     [tile
+      {:style    style/top-left
+       :size     size
+       :resource image-1}]
+     [tile
+      {:style    style/top-right
+       :size     size
+       :resource image-2}]
+     [tile
+      {:style    style/bottom-left
+       :size     size
+       :resource image-3}]]))
+
+(defn tile-container
+  [{:keys [images]}]
+  (let [num-images (count images)]
+    (case num-images
+      1 [tile
+         {:resource (first images)
+          :size     :xl}]
+      2 [two-tiles
+         {:images images
+          :size   :lg}]
+      3 [three-tiles
+         {:tiles images
+          :size  :md}]
+      (let [[first-three-images remaining-images] (split-at 3 images)]
+        [:<>
+         [three-tiles {:tiles first-three-images :size :md}]
+         [rn/view {:style style/tile-sub-container}
+          (case num-images
+            4 [tile
+               {:resource (nth images 3 nil)
+                :size     :md}]
+            5 [two-tiles
+               {:images (take 2 remaining-images)
+                :size   :sm}]
+            6 [three-tiles
+               {:tiles (take 3 remaining-images)
+                :size  :xs}]
+            7 [:<>
+               [three-tiles
+                {:tiles (take 3 remaining-images)
+                 :size  :xs}]
+               [tile
+                {:style    style/bottom-right
+                 :size     :xs
+                 :resource (nth remaining-images 3 nil)}]]
+            [:<>
+             [three-tiles
+              {:tiles (take 3 remaining-images)
+               :size  :xs}]
+             [remaining-tiles (- (count remaining-images) 3)]])]]))))
+
+(defn collectible
+  [{:keys [images]}]
+  [rn/view {:style style/tile-outer-container}
+   [rn/view {:style style/tile-inner-container}
```

Comment:
just wondering why it can't be only one container ?

Reply:
That's because the outer one sets the container of the whole component (176 x 176):
![image](https://user-images.githubusercontent.com/90291778/213250125-b8deabe1-21f8-466d-b025-24f44c96ccec.png)

and sets a padding of 8:
![image](https://user-images.githubusercontent.com/90291778/213250282-1cdf8ec6-3349-429d-a295-f64411116b14.png)

The inner one contains the tile or tiles, which are defined with an absolute position (styles `top-left`, `top-right`, `bottom-`...) to easliy create things like this:

top-left + bottom-right:
![image](https://user-images.githubusercontent.com/90291778/213251313-d6084334-bcee-4dcd-9ef3-df5afaccb547.png)

top-left + top-right + bottom-left
![image](https://user-images.githubusercontent.com/90291778/213251248-0dab18ea-dead-4bca-830b-f275100439cc.png)

If only the outer container exist and as I'm using absolute positions, the padding previously set (8px) would be ignored.

A solution could be add those paddings to the absolute styles, for example, instead of:
```clojure
(def bottom-left
  {:position :absolute
   :bottom   0
   :left     0})
```
do:
```clojure
(def bottom-left
  {:position :absolute
   :bottom   8
   :left     8})
   ```
   
but that wouldn't allow us to reuse the same styles in the smaller tiles and look the same as the design: 
   
![image](https://user-images.githubusercontent.com/90291778/213252772-91c840f7-31f4-42f0-a033-1722650fe397.png)
![image](https://user-images.githubusercontent.com/90291778/213252816-80dc3273-ead7-4cf6-8eff-4b26465b10bd.png)

Hope this is clear, let me know if any additional explanation is required!